app/src/main/java/com/marvel/marvel/main/model/ApiRetrofit.kt

```diff
@@ -0,0 +1,41 @@
+package com.marvel.marvel.main.model
+
+import com.marvel.marvel.main.model.pojos.Comics
+import okhttp3.OkHttpClient
+import okhttp3.logging.HttpLoggingInterceptor
+import retrofit2.Retrofit
+import retrofit2.adapter.rxjava.RxJavaCallAdapterFactory
+import retrofit2.converter.gson.GsonConverterFactory
+import rx.Observable
+import java.util.concurrent.TimeUnit
+
+
+class ApiRetrofit() : Api {
+    companion object {
+        private const val MAIN_URL = "https://gateway.marvel.com:443/"
+        private const val TIMEOUT_IN_SECONDS = 60L
+    }
+    private val api: Api
+
+    override val comics: Observable<Comics>
```

Comment:
It is behavior and different every time. It should be method, not property
https://blog.kotlin-academy.com/kotlin-should-i-define-function-or-property-6786951da909